File: dotNet/ClassLibrary/WinForm/ContactFormApp/ContactCore/LoginService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ContactCore
{
    public class LoginService
    {
        private string _userName;
        private string _password;

        public Boolean AuthenticateUser(string userId,string password)
        {
            LoginRepository loginRepository = new LoginRepository();
            _userName = loginRepository.UserName;
            _password = loginRepository.Password;

            if (_userName==userId && _password==password)
            {
                return true;
            }

            return false;
        }
    }
}

File: Ionic/TenantApp/src/Services/loginService.ts
import { TenantApiService } from "./tenantApiService";
import { Injectable } from "@angular/core";

@Injectable()
export class LoginService{
    constructor(private _tenantApiService:TenantApiService){

    }

    authenticateUser(username:string,password:string){
        return this._tenantApiService.AuthenticateUser(username,password);
    }
}